PWGMM/Mult/Tasks/multiplicityPbPb.cxx

```diff
@@ -155,6 +155,11 @@ struct multiplicityPbPb {
     for (auto& mcParticle : mcParticles) {
       ++MCparticleCounter;
       if (mcParticle.isPhysicalPrimary()) {
+        auto pdgparticle = pdg->GetParticle(mcParticle.pdgCode());
+        if (pdgparticle != nullptr)
+          if (pdgparticle->Charge() == 0)
```

Comment:
This value is stored as a floating point number, comparison of a floating point number to 0 is not guaranteed to be always correct. You can do something like `std::abs(pdgparticle->Charge()) > 2.` (note that the charge value is in units of |e|/3 and simulated particle stack may contain quarks).

Reply:
Hello, 
thank you, I need to distinguish charged and neutral particles. Will it work if I use std::abs(pdgparticle->Charge()) > 1?

Reply:
The value of `1` corresponds to 1/3 of the electron charge, so using condition `>1` would let particles with the charge of 2/3 of the electron charge into your sample. Asking for a particles with |charge| more than 2, i.e. `> 2/3|e|`, or, alternatively, skipping particles with |charge| less than 3. `< 1|e|`, is what you want.